crates/shared/src/signature_validator/arguments.rs

```diff
@@ -1,66 +0,0 @@
-use {
```

Comment:
This module had already been removed in #1842, but the code was accidentally left behind.